### PR TITLE
🚚 update: Add 132nd CatchUp summary

### PR DESCRIPTION
**Pull request by PranavDani**
Sorry for the delay. I don't have any particular excuse. I was out of my pace for a few days in a row.

### CODE REVIEW

**Comment by tusharnankani** [comment]
It's okay. All of us are struggling :'))

File: summary/sessions/132/content.adoc [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+Duration: 4 hrs
+
+==== Topics Discussed
+
+* link:https://twitter.com/saurabhdawaree[Saurabh Daware^] and link:https://twitter.com/PranavDani3[Pranav Dani^] distilled different types of jobs available based on need and skills.
+    ** We discussed the specifics of different domains like cloud, frontend, and backend.
+        *** There's no right time to make a switch in job roles. 
+        *** Changing roles can help prevent mental stagnation.
+    ** Darshit Suratwala talked about how banks work on different projects.
+* link:https://twitter.com/saurabhdawaree[Saurabh Daware^] shared how his team works with the front end by creating and maintaining react libraries for the organization.
```
Comment (tusharnankani) [inline summary/sessions/132/content.adoc:12]:
```suggestion
* link:https://twitter.com/saurabhdawaree[Saurabh Daware^] shared how his team works with the frontend by creating and maintaining React libraries for the organization.
```

File: summary/sessions/132/content.adoc [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+* We discussed potential improvements and similar implementations for _Vaulter_.
```
Comment (tusharnankani) [inline summary/sessions/132/content.adoc:13]:
```suggestion
* We discussed potential improvements and similar implementations for link:https://github.com/Chirag-And-Dheeraj/vaulter[Vaulter^] — A WIP project by link:https://twitter.com/DhiruCodes[Dheeraj Lalwani^] and link:https://twitter.com/_chiraglulla_[Chirag Lulla^].
```

File: summary/sessions/132/content.adoc [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            **** Check parity between each chunk and store it across multiple drives.
+        *** The working is similar to link:https://www.westerndigital.com/en-in/solutions/raid[RAID storage^].
+            **** Each system has a fault tolerance limit for drive failure.
+        *** Databases require high read speeds.
+            **** Differences in latency in the hierarchy of storage from the CPU cache to the hard drive range from nanoseconds to milliseconds.
+    ** How will the self-hosting happen?
+        *** The idea is to create a simple executable for users to run on their server of choice.
+            **** link:https://twitter.com/saurabhdawaree[Saurabh Daware^] suggested that he should first get the bare bones ready and then scale up from there.
+            **** By this time, work on optimizing the system and working on edge cases.
+    ** link:https://github.com/bashbaugh/cloud-storage-file-browser[GCP Storage Browser^], link:https://deta.space/discovery/@gyrooo/filebox[Filebox^] have similar implementations.
```
Comment (tusharnankani) [inline summary/sessions/132/content.adoc:31]:
```suggestion
    ** link:https://github.com/bashbaugh/cloud-storage-file-browser[GCP Storage Browser^] and link:https://deta.space/discovery/@gyrooo/filebox[Filebox^] have similar implementations.
```

File: summary/sessions/132/content.adoc [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    ** Anil Harwani talked about how file chunks are processed and saved in a drive.
+        *** How does block storage ensure that you never lose data? Is it storing all the data in one single hard drive?
+            **** Many cases implement sequential file chunking and saving.
+            **** Either store the data in a single hard drive or save multiple copies of it across multiple hard drives.
+            **** If stored on a single drive, chances of failure are high, so instead of that store it in multiple 1 GB chunks and create a hash function for addressing.
+            **** Every 1 gig chunk can have a parity algorithm to check the consistency of the data.
+            **** Check parity between each chunk and store it across multiple drives.
+        *** The working is similar to link:https://www.westerndigital.com/en-in/solutions/raid[RAID storage^].
+            **** Each system has a fault tolerance limit for drive failure.
+        *** Databases require high read speeds.
```
Comment (dheerajdlalwani) [inline summary/sessions/132/content.adoc:25]:
```suggestion
        *** Metadata databases require high read speeds.
```

File: summary/sessions/132/content.adoc [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    ** Most people can learn touch typing within 1 month if they had some experience with using musical instruments since that involves using multiple fingers simultaneously.
+* link:https://twitter.com/adityaoberai1[Aditya Oberai^] and Darshit Suratwala talked about potential MITM (Man in the Middle) attacks using link:https://www.wireshark.org[Wireshark^].
+    ** Wireshark is a packet analyzer that can be used to analyze network traffic, but most of the times the data is encrypted and hence it cannot alter requests with positive outcomes.
+    ** ISPs use similar ways to analyze traffic.
+
+==== Projects Showcased
+
+* link:https://twitter.com/DhiruCodes[Dheeraj Lalwani^] shared his progress on _Vaulter_, a self-hosted storage server.
+    ** It is a simple file-sharing and self-hosted storage server, inspired by an open-source web-based collection server and streamer, link:https://www.navidrome.org[Navidrome^] that he had earlier worked with.
+    ** link:https://github.com/Chirag-And-link:https://twitter.com/DhiruCodes[Dheeraj Lalwani^]/vaulter[GitHub Repository^]
```
Comment (tusharnankani) [inline summary/sessions/132/content.adoc:55]:
```suggestion
    ** link:https://github.com/Chirag-And-Dheeraj/vaulter[GitHub Repository^]
```

**Review by tusharnankani — COMMENTED** [review]
Done, let's go!

**Comment by HarshKapadia2** [comment]
Okay, I'm squashing and merging this PR as well. Thank you everyone!

**Pull request merged**